src/snowflake/connector/cpp/ArrowIterator/BinaryConverter.cpp

```diff
@@ -18,7 +18,7 @@ PyObject* BinaryConverter::toPyObject(int64_t rowIndex) const
 {
   if (m_array->IsValid(rowIndex))
   {
-    arrow::util::string_view sv = m_array->GetView(rowIndex);
+    std::string_view sv = m_array->GetView(rowIndex);
```

Comment:
why we switch from `arrow::util::string_view` to `std::string_view`?

Reply:
This seems to have been a change required by the underlying Arrow library after it upgraded to C++ 17.

Reply:
That's the new return value of 'GetView()' in PyArrow 10

https://arrow.apache.org/docs/cpp/api/array.html#_CPPv4NK5arrow15BaseBinaryArray7GetViewE7int64_t